### PR TITLE
Merge shared changes

### PR DESCRIPTION
**Pull request by millerds**
Thank you for your pull request!  Please provide the following information.

---

**Change Description**:
Get changes from the base taskpane repo and merge them here

1. **Do these changes impact any *npm scripts* commands (in package.json)?** (e.g., running 'npm run start')
No.

2. **Do these changes impact *VS Code debugging* options (launch.json)?**
No.

3. **Do these changes impact *template output*?** (e.g., add/remove file, update file location, update file contents)
Some manifest updates

4. **Do these changes impact *documentation*?** (e.g., a tutorial on https://docs.microsoft.com/en-us/office/dev/add-ins/overview/office-add-ins)
I don't believe so

If you answered yes to any of these please do the following:
    > Include 'Rick-Kirkham' in the review
    > Make sure the README file is correct

**Validation/testing performed**:
None

### CODE REVIEW

**Review by Adrian-MSFT — APPROVED** [review]

File: manifest.powerpoint.xml [hunk 1]
```diff
@@ -82,7 +82,7 @@
       </bt:ShortStrings>
       <bt:LongStrings>
         <bt:String id="GetStarted.Description" DefaultValue="Your sample add-in loaded successfully. Go to the HOME tab and click the 'Show Task Pane' button to get started." />
-        <bt:String id="TaskpaneButton.Tooltip" DefaultValue="Click to Show a Task Pane" />
+        <bt:String id="TaskpaneButton.Tooltip" DefaultValue="Click to Show a Taskpane" />
```
Comment (Rick-Kirkham) [inline manifest.powerpoint.xml:85]:
```suggestion
        <bt:String id="TaskpaneButton.Tooltip" DefaultValue="Click to Show a Task Pane" />
```

File: manifest.word.xml [hunk 2]
```diff
@@ -80,7 +80,7 @@
       </bt:ShortStrings>
       <bt:LongStrings>
         <bt:String id="GetStarted.Description" DefaultValue="Your sample add-in loaded successfully. Go to the HOME tab and click the 'Show Task Pane' button to get started." />
-        <bt:String id="TaskpaneButton.Tooltip" DefaultValue="Click to Show a Task Pane" />
+        <bt:String id="TaskpaneButton.Tooltip" DefaultValue="Click to Show a Taskpane" />
```
Comment (Rick-Kirkham) [inline manifest.word.xml:83]:
```suggestion
        <bt:String id="TaskpaneButton.Tooltip" DefaultValue="Click to Show a Task Pane" />
```

File: manifest.xml [hunk 3]
```diff
@@ -147,13 +147,13 @@
...
-        <bt:String id="TaskpaneButton.Label" DefaultValue="Show Task Pane"/>
+        <bt:String id="GetStarted.Title" DefaultValue="Get started with your sample add-in!" />
+        <bt:String id="CommandsGroup.Label" DefaultValue="Commands Group" />
+        <bt:String id="TaskpaneButton.Label" DefaultValue="Show Task Pane" />
       </bt:ShortStrings>
       <bt:LongStrings>
-        <bt:String id="GetStarted.Description" DefaultValue="Your sample add-in loaded successfully. Go to the HOME tab and click the 'Show Task Pane' button to get started."/>
-        <bt:String id="TaskpaneButton.Tooltip" DefaultValue="Click to Show a Task Pane"/>
+        <bt:String id="GetStarted.Description" DefaultValue="Your sample add-in loaded successfully. Go to the HOME tab and click the 'Show Task Pane' button to get started." />
+        <bt:String id="TaskpaneButton.Tooltip" DefaultValue="Click to Show a Taskpane" />
```
Comment (Rick-Kirkham) [inline manifest.xml:156]:
```suggestion
        <bt:String id="TaskpaneButton.Tooltip" DefaultValue="Click to Show a Task Pane" />
```

File: manifest.excel.xml [hunk 4]
```diff
@@ -80,7 +80,7 @@
       </bt:ShortStrings>
       <bt:LongStrings>
         <bt:String id="GetStarted.Description" DefaultValue="Your sample add-in loaded successfully. Go to the HOME tab and click the 'Show Task Pane' button to get started." />
-        <bt:String id="TaskpaneButton.Tooltip" DefaultValue="Click to Show a Task Pane" />
+        <bt:String id="TaskpaneButton.Tooltip" DefaultValue="Click to Show a Taskpane" />
```
Comment (Rick-Kirkham) [inline manifest.excel.xml:83]: Actually, it should be two words "Task Pane". It was wrong for years. It looks like at some point it was fixed and now this edit is reintroducing the mistake.

```suggestion
        <bt:String id="TaskpaneButton.Tooltip" DefaultValue="Click to Show a Task Pane" />
```

File: webpack.config.js [hunk 5]
```diff
@@ -34,9 +34,6 @@ module.exports = async (env, options) => {
             exclude: /node_modules/,
             use: {
               loader: "babel-loader",
```
Comment (Rick-Kirkham) [inline webpack.config.js:36]: Comma should be deleted too. 

```suggestion
              loader: "babel-loader"
```

**Review by Rick-Kirkham — CHANGES_REQUESTED** [review]

File: webpack.config.js (hunk 5 above)
Comment (millerds) [reply webpack.config.js:36]: Actually, prettier is setup to ask for commas to be there.  You'll notice that is the case in the rest of the file as well.  This seems to be the general practice except in the case of JSON which explicitly doesn't accept this.

**Review by Rick-Kirkham — APPROVED** [review]

**Pull request merged**